src/raster_programs.rs

```diff
@@ -0,0 +1,107 @@
+use crate::{
```

Comment:
Would be nice to move these into a sub-module `raster_programs::vrt` or similar.   Not a major concern, but this mod. might get cluttered if we bring in more of the programs like Warp, etc.